cmd/root_test.go

```diff
@@ -81,6 +82,20 @@ func invokeProxyCommand(args []string) (*Command, error) {
 	return c, err
 }
 
+func Test_UserAgentWithOperatorVersion(t *testing.T) {
```

Comment:
We've been omitting underscores in test names. Let's remove the underscore here.